attestation-service/bin/grpc-as/example.token.json

```diff
@@ -0,0 +1,47 @@
+{
+    "customized_claims": {
+        "test_key": "test_value"
+    },
+    "evaluation-reports": [
+        {
+            "evaluation-result": "{\"allow\":true}",
```

Comment:
does the evaluation result have to be a string? can't it be:
```json
"evaluation-result": {
  "allow": true
}
```

Reply:
This is the raw string output by OPA when using the `default_policy.rego`. I am afraid that we cannot assume what the result will look like. It is decided by the rego.

Reply:
we can specify what it needs to look like to be valid. and besides: the code already parses the OPA output and checks that it matches this structure.

Reply:
Let me parse the output directly as a JSON object. I think the document of policies can be added in another PR that also can promote the policy storage.